### ISSUE TITLE
trust_env=True of aiohttp.ClientSession should not print a warning log nor try to access each time a file

### ISSUE THREAD

**Issue by siom79** [aio-libs/aiohttp#3298]
## Long story short

When setting ``trust_env=True`` for ``aiohttp.ClientSession``, a warning like the following one is printed to the log if the file ``.netrc`` is missing.

```
2018-09-26 08:10:07 WARNING (MainThread) [aiohttp.client] could't find .netrc file
```
This should not happen as the intention is to only access environment variables. Beyond that, the file access should only happen once and not again and again for each instance of ``aiohttp.ClientSession``. This is for example the reason why pull request [16874](https://github.com/home-assistant/home-assistant/pull/16874) for [home-assistant](https://github.com/home-assistant/home-assistant) was rejected.

## Expected behaviour

Setting ``trust_env=True`` for ``aiohttp.ClientSession`` should not print a warning and should not try to access a file each time.

## Actual behaviour

If the file ``.netrc`` is missing, a warning is printed and for each instance of ``aiohttp.ClientSession`` the file check is repeated.

## Steps to reproduce

Create an instance of ``aiohttp.ClientSession`` with ``trust_env=True`` when there is no ``.netrc`` file.

## Your environment

aiohttp version 3.4.4, used as client, running on Ubuntu 18.04 with python 3.6.6

**Comment by asvetlov** [comment]
A warning is addressed by #3267 

Regarding caching `.netrc` parsing results -- it could confuse people.
If we add caching it should be an opt-in configurable option I guess.

**Comment by cooperlees** [comment]
I see fixes for this are all landed - When do we expect a >= 3.4.4 release as I'd love to enable this for bandersnatch (as linked).

**Comment by asvetlov** [comment]
No release date is scheduled.
This month perhaps.
